packages/destination-actions/src/destinations/personas-messaging-twilio/sendSms/index.ts

```diff
@@ -90,6 +90,12 @@ const action: ActionDefinition<Settings, Payload> = {
       description: 'Message to send',
       type: 'text',
       required: true
+    },
+    customArgs: {
+      label: 'Custom Args',
+      description: 'Additional custom args that will be opaquely sent back on webhook events',
```

Comment:
Might be worth spelling out "args" or using a different word that's more approachable for customers – soon action fields will be programmatically generating docs in our catalog docsite.

```suggestion
      description: 'Additional custom arguments that will be opaquely sent back on webhook events',
```

Reply:
Should I spell out the label, too? I wasn't sure where that was used.

Reply:
TBH I'm not sure how this is rendered by Personas UI – but in the main catalog and action editor it looks like this: 
![Screen Shot 2021-09-10 at 3 06 01 PM](https://user-images.githubusercontent.com/710752/132911440-8683685d-a9ff-48dd-8ba4-e48a3478a3cf.png)

This screenshot is of the Slack action-destination: 
https://github.com/segmentio/action-destinations/blob/5bd7ff8b79e903ecc8af2ec2b2b12f21f23bbbd2/packages/destination-actions/src/destinations/slack/postToChannel/index.ts#L8-L40

So you can see how the label + description are displayed to users.

Reply:
Gotcha. I think based off of that and this view from testing the SendGrid action it makes more sense to have the label be readable.

![image](https://user-images.githubusercontent.com/87994541/132912371-84708fc8-4563-4d4c-bfe0-f96dae2cfb9f.png)